### ISSUE TITLE
Typo in rgb2lab transform

### ISSUE THREAD

**Issue by Vlad-C-Crisan** [EIDOSLAB/torchstain#50]
**Describe the bug**
In [THIS LINE](https://github.com/EIDOSLAB/torchstain/blob/3a3eb2c8af2662b4a001cf134f9a1bd24e0bdba7/torchstain/torch/utils/rgb2lab.py#L28) from rgb2lab implementation, the value `166` should be `116`.

**Expected behavior**
The line should be changed to `arr.masked_scatter_(not_mask, 7.787 * torch.masked_select(arr, not_mask) + 16 / 166)` to match numpy implementation (e.g. [THIS one](https://github.com/scikit-image/scikit-image/blob/49a98547e909697f6ee89ec70691c7cad7eeb823/skimage/color/colorconv.py#L1082) from skimage)

**Comment by andreped** [comment]
Good catch, @Vlad-C-Crisan! Surprising that this has not yet been observed.

I can make a PR.

**Comment by andreped** [comment]
Made PR: https://github.com/EIDOSLAB/torchstain/pull/51
